sha1-name.c

```diff
@@ -1729,7 +1783,7 @@ static int get_oid_with_context_1(const char *name,
 					diagnose_invalid_oid_path(prefix,
 								   filename,
 								   &tree_oid,
-								   name, len);
+								   name, len, 0);
```

Comment:
That's surprising. Why pass `0` even if the `flags` bitfield contains `GET_OID_GENTLY`?